charts/sn-platform/templates/presto/presto-destinationrule.yaml

```diff
@@ -0,0 +1,60 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if and .Values.components.sql_worker .Values.istio.enabled }}
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: "{{ template "presto.coordinator" . }}"
+spec:
+  host: "{{ template "presto.hostname" . }}"
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+---
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: "{{ template "presto.coordinator" . }}-headless"
```

Comment:
I can't find headless service for service?

Reply:
Pod dns records can be generated by Statefulset without headless Service, following the rule of `${podName}.${.spec.serviceName}.${namespace}.svc.cluster.local`.

ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#stable-network-id